PWGGA/GammaConv/AliAnalysisTaskGammaCaloMergedML.cxx

```diff
@@ -504,8 +510,16 @@ AliAnalysisTaskGammaCaloMergedML::AliAnalysisTaskGammaCaloMergedML(const char *n
   
   // Define output slots here
   DefineOutput(1, TList::Class());
-  //DefineOutput(2, TTree::Class());
-  //DefineOutput(3, TTree::Class());
+  DefineOutput(2, TTree::Class());
```

Comment:
another general comment here. I think creating multiple trees in the same file is a bit unconventional. I think what one would usually do is fill into the same tree and have some integer that identifies the kind of true particle (e.g. PDG coe or special integer for background etc, 0 if not a real particle ...)